src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -82,12 +82,15 @@
   }
 
   /**
-   * Create a client builder with a basic string transcoder.
-   * @param charset the charser to use for encoding/decoding strings
+   * Create a client builder with a basic string transcoder using the UTF-8
+   * Charset. To build a client with a different charset, provide a StringTranscoder
+   * constructed with a different charset in the public MemcacheClientBuilder
```

Comment:
I guess it's a matter of opinion how much you want to discourage custom charsets. I don't think that the documentation hint I added necessarily indicates a problem, but take your point.